### ISSUE TITLE
Add "new" option

### ISSUE THREAD

**Issue by derekdreery** [rw251/getset#14]
When working on an existing saved term set, it is not possible to start again on a new term set. This is a problem if the user wants to create multiple named term sets in a single session.

**Comment by rw251** [comment]
Agreed. My current workaround is just to clear all the codes and refresh the page.

**Comment by derekdreery** [comment]
The name is [bikesheddable]. I suggested "new" because that matches e.g. MS Office. Other possibilities are things like "clear".

[bikesheddable]: https://en.wiktionary.org/wiki/bikeshed
